### ISSUE TITLE
[TooltipHover]: text in tooltip will overflow after reached certain length

### ISSUE THREAD

**Issue by ziheng** [carbon-design-system/carbon-addons-ics#50]
## Detailed description

Describe in detail the issue you're having. Is this a feature request (new component, new icon), a bug, or a general issue?
![image 1](https://user-images.githubusercontent.com/9008867/39571460-711611e8-4ec3-11e8-87ce-923cd63884dc.png)

> Is this issue related to a specific component?
TooltipHover

> What did you expect to happen? What happened instead? What would you like to see changed?
The text should not overflow, after text input reaches certain length it should go to a second line.

> What browser are you working in?
Chrome

> What version of the Carbon Design System and carbon-addons-ics are you using?
2.0.0

**Comment by j1mie** [comment]
Thanks for posting @ziheng and apologies for the delay - I will try to replicate this issue and have this prioritised for our upcoming sprint.

**Comment by ZacharyStair** [comment]
@j1mie yup, thats why we switched to using the [complex tooltip with the `openOnHover` flag](http://ics.carbondesignsystem.com/?selectedKind=Components%7CTooltip&selectedStory=Complex%20tooltip%20on%20hover&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel)
